protocols/ping/CHANGELOG.md

```diff
@@ -2,10 +2,16 @@
 
 - Raise MSRV to 1.65.
   See [PR 3715].
+
 - Remove deprecated items. See [PR 3702].
 
+- Don't close connections on ping failures.
```

Comment:
```suggestion
- Don't close connections on ping failures.
  In case users want to disconnect on ping failure, they need to do so manually via `Swarm::disconnect_peer_id`.
```

Writing this out, I don't think there is a way for a user to close a specific connection through the `Swarm` API, right? How would one close a faulty connection to a remote when one already has a new connection?

Reply:
That is a good point, I guess we should add that API.

Reply:
```suggestion
- Don't close connections on ping failures.
  To restore the previous behaviour, users should call `Swarm::close_connection` upon receiving a `ping::Event` with a `ping::Failure`.
```